lib/jazzy/grouper.rb

```diff
@@ -69,7 +69,42 @@ def self.group_custom_categories(docs, doc_index)
         children.each.with_index { |child, i| child.nav_order = i }
         make_group(children, category['name'], '')
       end
-      [group.compact, docs]
+
+      globals_grouped_by_file = self.group_globals_by_file(docs, doc_index)
```

Comment:
Hi @johnfairh, should I place this under a new command line argument or something?

You recommended including a `glob/wildcard/regexp`, could you please elaborate a little more on what such a regexp would be used to filtered? Would it be used to determine whether constants in a file should be affected by this logic or not?

I can make that change in a following commit.

Reply:
The 'glob' comment was in response to your example of `"children": [file:GlobalUtils.h"` which is not the feature you ended up going for.

The default behaviour mustn't change because of existing users, new behaviour can be behind a CLI option.